packages/lesswrong/lib/collections/posts/schema.ts

```diff
@@ -542,12 +542,14 @@ const schema = {
     type: "Comment",
     graphQLtype: "Comment",
     viewableBy: ['guests'],
-    resolver: async (post) => {
+    resolver: async (post, args, { currentUser, TagRels }) => {
```

Comment:
@Discordius this still references TagRels up here, I think just unnecessarily

Reply:
(also fixed)